docs/book/utils/markerdocs/html.go

```diff
@@ -126,6 +132,7 @@ type dataAttr struct {
 	Value string
 }
 
+// ToAttrs implements Attrs
 func (d dataAttr) ToAttrs() string {
```

Comment:
Could we give a little more info here? WDYT about add why/when it can be used?

Reply:
See https://github.com/kubernetes-sigs/kubebuilder/pull/1368#discussion_r383882586